Instagram/Models/Post.m

```diff
@@ -0,0 +1,55 @@
+//
+//  Post.m
+//  Instagram
+//
+//  Created by Sarah Wang on 7/7/21.
+//
+
+
+#import "Post.h"
+
+@implementation Post
+
+@dynamic postID;
+@dynamic userID;
+@dynamic author;
+@dynamic caption;
+@dynamic image;
+@dynamic likeCount;
+@dynamic commentCount;
+
+
++ (nonnull NSString *)parseClassName{
+    return @"Post";
+}
+
++ (void) postUserImage: ( UIImage * _Nullable )image withCaption: ( NSString * _Nullable )caption withCompletion: (PFBooleanResultBlock  _Nullable)completion {
+    
+    Post *newPost = [Post new];
+    newPost.image = [self getPFFileFromImage:image];
+    newPost.author = [PFUser currentUser];
+    newPost.caption = caption;
+    newPost.likeCount = @(0);
```

Comment:
Could you just do `@0` ? This makes it an NSNumber which is basically the same as what you're doing which is saying 'treat this int as an object' (pretty sure that just makes it an NSNumber but don't quote me on that).